plugins/UM3NetworkPrinting/PrinterInfoBlock.qml

```diff
@@ -165,6 +166,7 @@ Rectangle
                 anchors.right: printProgressArea.left
                 anchors.rightMargin: UM.Theme.getSize("default_margin").width
                 color: emphasisColor
+                opacity: printer != null && printer.status === "unreachable" ? 0.3 : 1
```

Comment:
In what cases could the printer.status be anything but a string? eg; Why is it required to use strict comparison. As far as I see, it's actually better not to use strict comparison, as in quite a few cases the values can come from python (and thus we should assume duck typing, so if it pretends well enough to be a string, its fine!)